src/components/Card/Footer/CardFooter/CardFooter.tsx

```diff
@@ -39,17 +38,20 @@ const CardFooter: FunctionComponent<IProps> = ({
           </Card>
         </Grid>
         <Grid item xs={4}>
-          <FilterTodos
-            filterTodos={filterTodos}
+          <FilterTodosToggle
+            todosFilter={todosFilter}
             onChange={handleFilterChange}
           />
         </Grid>
         <Grid item xs={4}>
-          <ClearCompleted onClear={clearAllCompleted} />
+          {/* QUESTION: */}
+          {/* ClearCompletedButton will be rerendered every time todos array is changed, because onClear useCallback has to have todos in his deps array */}
+          {/* so is it worth it to use React.memo here? I'm not really sure */}
+          <ClearCompletedButton onClear={clearAllCompleted} />
```

Comment:
CardFooter is not that high in the dom tree but it still contains few components beneath it, but in the end memo won't give you much better performance, but react.memo here will compare references things that are not heavy to compare.

so in this case I would have used memo